drf_braces/serializers/enforce_validation_serializer.py

```diff
@@ -31,13 +31,26 @@ def run_validation(self, data=empty):
             if must_validate_fields is None or field_name in must_validate_fields:
                 raise
             else:
+                self.capture_failed_field(field_name, data, e)
                 raise fields.SkipField(
                     'This field "{}" is being skipped as per enforce validation logic.'
                     ''.format(field_name)
                 )
 
+    def capture_failed_field(self, field_name, field_data, exception):
+        """Hook for capturing invalid fields. This is used to track which fields have been skipped.
```

Comment:
nitpik. rest of the files have a convention of starting docstring on new line. can we match?

Reply:
Yep